### DISCUSSION TITLE
Does K3S support Postgres 16?

### DISCUSSION THREAD

**Discussion by mike12806** [k3s-io/k3s#10912] [Q&A]
[Docs](https://docs.k3s.io/datastore) say it's only supported up to 15.4 atm, but curious if anyone is using postgres 16

**Comment by brandond** [comment]
I don't think we've tested it yet - not here or in kine itself:
https://github.com/k3s-io/kine/blob/v0.12.2/scripts/test-run-postgres#L25-L29

cc @ShylajaDevadiga
